Build Error when using "documentLang" in TypeScript

Hi,
i don't know why i didn't get this error earlier, but using v0.17.0-canary.1 and upwards if i use `next-translate/documentLang` in the `_document.tsx` i get the following error on build:

**Console Output**:
[console.txt](https://github.com/vinissimus/next-translate/files/4702786/console.txt)

If i remove the `documentLang` this error doesn't appear.

Seems the important part is `Automatically optimizing pages .Error: Cannot find module '/Users/userx/projects/own/project/packages/project-web/i18n.json'`.

Please let me know if i can/should provide more information :)

--- a/src/documentLang.js
+++ b/src/documentLang.js
@@ -1,12 +1,15 @@
 const path = require('path')
 const fs = require('fs')
 
-export default function documentLang({ __NEXT_DATA__ }) {
-  const configDir = path.join(process.cwd(), 'i18n.json')
-  const i = JSON.parse(fs.readFileSync(configDir))
+export default function documentLang({ __NEXT_DATA__ }, config) {
+  if (!config) {
+    const configDir = path.join(process.cwd(), 'i18n.json')
+    config = JSON.parse(fs.readFileSync(configDir))
+  }
+
   const { page } = __NEXT_DATA__
   const [, langQuery] = page.split('/')
-  const lang = i.allLanguages.find((l) => l === langQuery)
+  const lang = config.allLanguages.find((l) => l === langQuery)
 
-  return lang || i.defaultLanguage
+  return lang || config.defaultLanguage
 }

